src/lib.rs

```diff
@@ -677,6 +677,20 @@ macro_rules! ok {
     };
 }
 
+/// Workaround because `?` is not (yet) available in const context.
+///
+/// FIXME: This is a temporary macro to simplify `try_opt!(ok!(result))`
+#[macro_export]
+#[doc(hidden)]
+macro_rules! try_opt_ok {
```

Comment:
I don't care much for this macro, but it should be just temporary like `ok!` and be cone in a couple of weeks/months.